### PR TITLE
LPS-200593 - Fix Workflow

### PR DESCRIPTION
**Pull request by thaiscabrals**
**Ticket:** [LPS-200593](https://liferay.atlassian.net/browse/LPS-200593)
**Test:** MyWorkflowTasks#CanAssignTaskToUserWithAccountRole

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 1]
```diff
@@ -610,7 +610,7 @@ definition {
 
 		Workflow.viewTaskAssignedToMyRoles(
 			workflowAssetTitle = "Order",
-			workflowAssetType = "Commerce Order",
+			workflowAssetType = "Order",
```
Comment (thaiscabrals) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:613]: The assignment has changed. Now it is no longer assigned to the **_Commerce Order_**, only to the **_Order_**.

**Review by evanilson-santana — APPROVED** [review]
LGTM :tada:

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**